pontoon/base/views.py

```diff
@@ -788,6 +788,26 @@ def download_translation_memory(request, locale, slug, filename):
     return response
 
 
+@utils.require_AJAX
+def user_data(request):
+    user = request.user
+
+    if not user.is_authenticated:
+        return JsonResponse({
+            'is_authenticated': False,
+        })
+
+    return JsonResponse({
+        'is_authenticated': True,
+        'id': user.id,
+        'email': user.email,
+        'display_name': user.display_name,
+        'manager_for_locales': list(user.managed_locales.values_list('code', flat=True)),
+        'translator_for_locales': list(user.translated_locales.values_list('code', flat=True)),
+        'translator_for_projects': user.translated_projects,
```

Comment:
There's more user data passed to `translate.html` and `translate.js`:
* `data-force-suggestions`
* `data-tour-status`
* `data-locales-order`

Shouldn't we include that, too?

There are two other pieces of data, but it doesn't look like they are used:
* `data-username`
* `data-manager`

Reply:
SGTM